### ISSUE TITLE
Layout styles

### ISSUE THREAD

**Issue by bronze** [RafidMuhymin/astro-imagetools#14]
Is there a way to stop the component inserting inline css with the image?
I'm using this to load a bg image, the `layout=fill` has worked, just wanted to clean it up a bit.

Something like `layout="none"` maybe?

**Comment by RafidMuhymin** [comment]
👍 Sounds interesting! I will take a look at this after the `v0.1.0` (planning to do it today) release.

And two questions for you too, 😅. `layout=fill` is different from `layout=none`. How did it work for your use case?

And how are loading a background image using the component? It'll help me to implement background image optimization in the plugin.

**Comment by bronze** [comment]
I'm just inserting the image component with absolute positioning and object-fit behind my content. [[Github link]](https://github.com/bronze/digitalcanvas-com-br/blob/master/src/pages/index.astro) [[Live Link]](https://www.digitalcanvas.com.br/)

I was a bit surprised after inserting the image that it didnt keep its full page proportions (hadnt seen the inline style yet).
The `fill` essentially worked the same, just had it all done before on css.

**Comment by bronze** [comment]
Hi again.
Wanted to update astro and everything else to the latest version. Found out now image-tools is also inserting `style="position: relative; display: inline-block;"` into the `picture` element.
Can i disable this too? Have no inline styles?

**Comment by RafidMuhymin** [comment]
Yeah, you are correct @bronze, they have been added to fix a few issues and for the transition between the placeholder and the image. And now, it's not impossible but doing this has become more complex than before.

**Comment by bronze** [comment]
I was using the `<Picture>` element as a background image and had to use `position: absolute !important;` on my image to make it work.
i could probably just use the `BackgroundPicture` now, but i just wanted to have a clean output and style things myself.

**Comment by RafidMuhymin** [comment]
That makes sense. So, does the title `Layout styles` still make sense? I mean both the layout styles, other inline styles, and the CSS styles can cause issues while manually styling the picture.

**Comment by bronze** [comment]
"Built-in styles" might be more inclusive. 

Do you think this could be an option or maybe a simplified component? Maybe this can be achieved via the API already.

**Comment by RafidMuhymin** [comment]
The APIs that are available currently return HTML strings. So, if you want to do this you'll have to manually manipulate them. But I am serious about this feature and I will try to work on this ASAP!

**Comment by bronze** [comment]
Hi @RafidMuhymin ive tried just passing a "nocss" flag along with the img code to disable any css (currently working on Img and Picture. Wanted to know what you think.
![image](https://user-images.githubusercontent.com/170220/170811754-25df51bb-54ae-4013-815e-f2ed50b00b1d.png)
(im displaying the `nocss=true` for debugging purposes)

https://github.com/bronze/astro-imagetools/tree/nocss

**Comment by IanVS** [comment]
I'm also struggling a little bit here, due to the `display: inline-block;` being set inline on the `Img` component.  I want to center the image, and I'm trying to use a css class to set `display: block; margin: 0 auto;`, but the inline styles are winning out.  I can add it as a `style` in `attributes.img`, but that just feels like escalating the specificity wars, and doesn't feel very scalable.

**Comment by LushawnDev** [comment]
Hey, I'd love to know if there is any update on this? I am often finding myself battling with the inline styles in a CSS war to make the layouts work as I want.

Did you find a workaround @IanVS ?
